### DISCUSSION TITLE
central support

### DISCUSSION THREAD

**Discussion by sadden3194** [chatwoot/chatwoot#5818] [Show and tell]
** Englisch Version **
Hi all!

I would like to draw attention to this forum => [www.chatwoot-community.com](http://www.chatwoot-community.com/)
It's new and should serve as a central point for Chatwoot support, instead of multiple platforms (Github, Discord, ...).

These platforms are still available - however, I would find a central platform better and clearer.
I would be happy if you would support this idea. Support will be available in english and german.

Thank you very much - let's make chatwoot better together!

++ For this platform I got permission from Chatwoot support (Contact: Nithin David Thomas) ++

** Deutsche Version **
Hallo zusammen!

Ich möchte gerne auf dieses Forum aufmerksam machen => [www.chatwoot-community.com](http://www.chatwoot-community.com/)
Es ist neu und soll als zentrale Anlaufstelle für Chatwoot-Support dienen, anstelle von mehreren Plattformen (Github, Discord, ...).

Diese Plattformen stehen zwar weiterhin zur Verfügung - ich fände aber eine zentrale Plattform besser und übersichtlicher.
Ich würde mich freuen, wenn ihr diese Idee unterstützen würdet. Der Support wird in englisch und deutsch verfügbar sein.

Vielen Dank - lassen Sie uns gemeinsam Chatwoot besser machen!

++ Für diese Plattform habe ich die Erlaubnis vom Chatwoot-Support (Kontakt: Nithin David Thomas) ++

**Comment by pranavrajs** [comment]
Hey, Thanks for the interest, appreciate the effort. However, we cannot endorse the forum since we are not administrating it. Happy to help if you have any questions.
